### DISCUSSION TITLE
[Bug]: camera export name not matching

### DISCUSSION THREAD

**Discussion by GiorgioUghini** [blakeblackshear/frigate#15243] [Report a Bug]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

Given this documentation:
`POST /api/export/<camera>/start/<start-timestamp>/end/<end-timestamp>
Export recordings from start-timestamp to end-timestamp for camera as a single mp4 file. These recordings will be exported to the /media/frigate/exports folder.

It is also possible to export this recording as a time-lapse using the "playback" key in the json body, or specify a custom export filename, using the "name" key.

Optional Body:

{
  "playback": "realtime", // playback factor: realtime or timelapse_25x
  "name": "custom export name" // override the default export filename with a custom name
}`

### Steps to reproduce

If I send a request like this:
`curl -X POST \
  http://127.0.0.1:5000/api/export/camera_1/start/1732818565/end/2000000000 \
  -H "Content-Type: application/json" \
  -d '{"playback": "realtime", "name": "camera_1_2024_11_28_19_30__gZj2F8Kywk.mp4"}'
{"message":"Starting export of recording.","success":true}`

I would expect the file to be named camera_1_2024_11_28_19_30__gZj2F8Kywk.mp4. But instead, it is named with the id.
In fact look at this output:
`curl localhost:5000/api/exports
[{"camera":"camera_1","date":1732818867,"id":"camera_1_t083j6","in_progress":false,"name":"camera_1_2024_11_28_19_30__gZj2F8Kywk.mp4","thumb_path":"/media/frigate/clips/export/camera_1_t083j6.webp","video_path":"/media/frigate/exports/camera_1_t083j6.mp4"}]`

### Version

0.14.1-f4f3cfa

### Frigate config file

```yaml
Not needed
```

### docker-compose file or Docker CLI command

```yaml
Not needed
```

### Relevant Frigate log output

```shell
Not needed
```

### Relevant go2rtc log output

```shell
Not needed
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Not needed

### Screenshots of the Frigate UI's System metrics pages

Not needed

**Comment by NickM-27** [comment]
This is not a bug, the name refers to the name field not the file name

**Comment by GiorgioUghini** [comment]
Ehm, that is exactly why I posted the API description from the official Frigate page.
As written in the post above, the API specification clearly states: `"name": "custom export name" // override the default export **filename** with a custom name`.
To me, this bug report should not be closed so quickly.

**Reply by NickM-27** [reply]
That's how it used to work but it was changed in 0.14

In any case, the documentation has already been updated to reflect this https://deploy-preview-13787--frigate-docs.netlify.app/integrations/api/export-recording-export-camera-name-start-start-time-end-end-time-post
